setup.cfg

```diff
@@ -1,13 +1,14 @@
 [metadata]
-name = panda3d-gltf
+name = panda3d-glTF
```

Comment:
Name change?  Most packages in the Python ecosystem seem to be lowercase-named.

Reply:
It is, but by that logic shouldn't it be Panda3D-glTF, which looks even weirder as a package name?

Reply:
Ah, the remark was just an aside for another reason why I didn't change it initially.  I personally don't really care.